playbooks/roles/prometheus/meta/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+dependencies:
+  - name: ernestas-poskus.prometheus
+    prometheus_node_exporter_install: false
+  - name: nginx-proxy
+    NGINX_PROXY_CONFIG_PATH: /etc/nginx/sites-available/prometheus
+    NGINX_PROXY_PUBLIC_PORT: 443
+    NGINX_PROXY_INTERNAL_PORT: 9090
+    NGINX_PROXY_USERNAME: "{{ PROMETHEUS_BASIC_AUTH_USERNAME }}"
+    NGINX_PROXY_PASSWORD: "{{ PROMETHEUS_BASIC_AUTH_PASSWORD }}"
+  - name: nginx-proxy
+    NGINX_PROXY_CONFIG_PATH: /etc/nginx/sites-available/grafana
+    NGINX_PROXY_PUBLIC_PORT: 3443
+    NGINX_PROXY_INTERNAL_PORT: 3000
+    NGINX_PROXY_USERNAME: "{{ PROMETHEUS_BASIC_AUTH_USERNAME }}"
+    NGINX_PROXY_PASSWORD: "{{ PROMETHEUS_BASIC_AUTH_PASSWORD }}"
```

Comment:
I do think we should just use the Grafana login screen here instead of having nginx basic auth, as you suggested from the ansible-secrets PR. If I understand correctly, we can configure the user then in the future through Grafana rather than having to run the nginx role here to update it.

Reply:
Yeah, in hidnsight this seems like the far better option.  Otherwise the password always has to be changed in sync, which is rather unexpected and confusing.